pkg/api/customization/vsphere/listers.go

```diff
@@ -0,0 +1,401 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vapi/library"
+	"github.com/vmware/govmomi/vapi/rest"
+	"github.com/vmware/govmomi/vapi/tags"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+func soapLister(ctx context.Context, fieldName string, cc *v1.Secret, dc string) ([]string, error) {
+	finder, err := getSoapFinder(ctx, cc, dc)
+	if err != nil {
+		return nil, err
+	}
+
+	var data []string
+	switch fieldName {
+	case "clusters":
+		data, err = listClusters(ctx, finder) //tbd
```

Comment:
is this still tbd ? If so can you add some more info on what it is we are waiting for

Reply:
nope, it's not tbd any more. it's complete. ill remove